src/shared/registerButton/RegisterButton.jsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import "./RegisterButton.css";
+import { Link } from "react-router-dom";
+
+const RegisterButton = ({ to, onClick, children }) => {
```

Comment:
const AuthButton = ({ to, onClick, children, displayText}).

Reply:
completed